### ISSUE TITLE
LM4 Finalisations

### ISSUE THREAD

**Issue by lokka30** [ArcanePlugins/LevelledMobs#427]
> On hold!

- chunk kill count settings
  - `maximum-death-in-chunk-threshold:`
  - `max-adjacent-chunks:`
  - `chunk-max-cooldown-seconds:`
  - `disable-vanilla-drops-on-chunk-max:`
- spawner particle settings
  - `spawner-particles`
  - `spawner-particles-count`
- add action to set base values of attributes
- add LM4 equivalents for some LM3 placeholders – [docs](https://github.com/ArcanePlugins/LevelledMobs/wiki/Documentation---Apply-Settings#hard-coded-placeholders)
  - `%levelledmobs_killed-by%`
  - `%death_nametag%`
  - `%tiered%`
  - `%wg_region%`
  - `%world%`
  - `%location%` `%x%` `%y%` `%z%`
- add LM4 equivalents for LM3 ApplySettings 
  - `sunlight-intensity:`
    - in LM4 we will allow damage multipliers for any DamageCause instead
  - `lock-entity`
  - unsure if these will be added:
    - `nametag-placeholder-unlevelled: nametag-placeholder-levelled:`

- add migration code for customdrops.yml (LM3 -> LM4 file conversion). (@lokka30)
- add decorations to all external files (@UltimaOath and @lokka30 )
- Update README.md since LM3's and LM4's are different, stuff needs to be merged (@lokka30)
- User testing. In parallel, work on documentation (@lokka30 and @UltimaOath)
- Make LevelledMobs 4 update poster/banner, like what was used on LM3 update. (@UltimaOath)
- Release 😄 
- Publish LM4 to a Maven repository
- Update LM Items to have v4 support (@stumper66)

**Comment by stumper66** [comment]
This is no longer applicable.
